orchestrator/careplancontributor/service.go

```diff
@@ -35,6 +35,7 @@ const basePath = "/cpc"
 
 // The care plan header key may be provided as X-SCP-Context but will be changed due to the Go http client canonicalization
 const carePlanURLHeaderKey = "X-Scp-Context"
+const carePlanServiceURLHeaderKey = "X-Cps-Url"
```

Comment:
SCP and CPS look terribly similar, some comments might help;

```suggestion
// carePlanURLHeaderKey specifies the HTTP request header used to specify the SCP context, which is a reference to a FHIR CarePlan. Authorization is evaluated according to this CarePlan.
// The header may also be provided as X-SCP-Context, which will be canonicalized to X-Scp-Context by the Golang HTTP client.
const carePlanURLHeaderKey = "X-Scp-Context"
// carePlanServiceURLHeaderKey specifies the HTTP request header to specify the FHIR base URL of the Care Plan Service the client wishes to invoke.
const carePlanServiceURLHeaderKey = "X-Cps-Url"
```